### PR TITLE
Enhancement: Issue39

### PR DESCRIPTION
**Pull request by apiraam96**
Issue 39: Added copy button to protein sequence and DNA sequence. Note that it copies the full DNA sequence presented.

https://github.com/BioAnalyticResource/ePlant/assets/103979336/e07fb9c9-4501-4d50-9b39-aaa22508f6cb

### CODE REVIEW

**Review by mwkyuen — APPROVED** [review]
LGTM!

File: Eplant/views/GeneInfoView/component.tsx [hunk 1]
```diff
@@ -235,6 +248,22 @@ export default function GeneInfoViewer({
...
+              <Snackbar
+                anchorOrigin={
+                  {
+                    vertical: 'top',
+                    horizontal: 'center'
+                  }
+                }
+                open={snackBarOpen}
+                onClose={() => setSnackBarOpen(false)}
+                autoHideDuration={2000}
```
Comment (mwkyuen) [inline Eplant/views/GeneInfoView/component.tsx:263]: Is this in seconds?

Comment (VinLau) [reply Eplant/views/GeneInfoView/component.tsx:263]: Should be milliseconds.

Comment (apiraam96) [reply Eplant/views/GeneInfoView/component.tsx:263]: Yeah that's in milliseconds

**Comment by alexwaeseperlman** [comment]
This is ready to merge, so feel free to do that. I had some comments that might be useful:
1. I think making the copy button color secondary would look better
2. I prefer pull requests with more descriptive titles. It's better to give a short description and then link to the issue by writing #39 (`#39` will generate the link)
3. If you want to link a PR to a commit more literally, write a commit message that says something like `fix #39`, and [github will automatically close the issue once the commit is merged](https://docs.github.com/en/issues/tracking-your-work-with-issues/linking-a-pull-request-to-an-issue#linking-a-pull-request-to-an-issue-using-a-keyword)

**Pull request merged**